src/sdl2/mixer.nim

```diff
@@ -432,7 +433,7 @@ proc getMusicDecoder*(index: cint): cstring {.importc: "Mix_GetMusicDecoder".}
   ## This string is owned by the SDL_mixer library, do not modify or free it.
   ## It is valid until you call `mixer.closeAudio()` the final time.
 
-proc getMusicType*(music: ptr Music): MusicType {.importc: "Mix_GetMusicType".}
+proc getType*(music: ptr Music): MusicType {.importc: "Mix_GetMusicType".}
```

Comment:
Breaking change, the old `getMusicType` must remain too.

Reply:
I didn't mean to. Fixed.